1_Mechanics/02_GMGuide.md

```diff
@@ -280,8 +299,21 @@ As a Game Master (GM), you will set up various challenges wherein you or a
 Player will draw a Target Card (TC), and the other draws against that card to
 make a Check. The level of difficulty of a Check is set by the Difficulty
 Range (DR), which is the distance between pip values that will still count as
-a success. The Target Card System is circular, so an Ace is only 1 away from
-both a King and a 2. 
+a success. When thinking about the Target Card System, it may be helpful to
+consider a bullseye, where the Target Card is the center of the bullseye and
+the DR is the number of circles away from the bullseye you can hit and still
+be successful. 
+
+One of the initial barriers a new GM must overcome is helping the Players
+understand the Target Card system. As Players get more familiar with the
+system, speed of play increases, and it is then easier for them to understand
+more complex mechanics. Using the [Circular TC-DR Tool](.../1_Mechanics/Circlular_TC-DR_Tool/1_Assembled.PNG) is a helpful visual
```

Comment:
This might have been my mistake. For relative references
- One dot means 'here'
- Two dots means 'up'
- Slash means folder
```suggestion
more complex mechanics. Using the [Circular TC-DR Tool](./Circlular_TC-DR_Tool/1_Assembled.PNG) is a helpful visual
```